model_templates/inference/r_unstructured/README.md

```diff
@@ -0,0 +1,121 @@
+## R Unstructured Inference Model Template
+
+This model is intended to work with the [R Drop-In Environment](../../../public_dropin_environments/r_lang/).
+or any other python environment as it doesn't require any specific packages.
+
+The data is just a text file [unstructured_data.txt](../../../tests/testdata/unstructured_data.txt)
+
+This model doesn't use any artifacts, but only a `custom.R` which implements the `score_unstructured` method and performs word count on input data.
+
+## Instructions
+Create a new custom model with these files and use the Python Drop-In Environment with it
+
+### To run locally using 'drum'
+Paths are relative to `./datarobot-user-models`:   
+`drum score --code-dir model_templates/inference/r_unstructured --target-type unstructured --input tests/testdata/unstructured_data.txt`
+
+#### Running with additional params
+THe goal of this example is to show how to control types of the values passed into `score_unstructured` hook.
+
+Consider the following signature: `score_unstructured <- function(model, data, query, ...)`
+
+##### Example 1
+Command:   
+`drum score --code-dir model_templates/inference/r_unstructured --target-type unstructured --input tests/testdata/unstructured_data.txt --verbose`
+
+Output:
+```
+[1] "Model: " "dummy"
+  
+[[1]]
+[1] "Incoming content type params: "
+
+$mimetype
+[1] "text/plain"
+
+$charset
+[1] "utf8"
+
+[1] "Incoming data type: " "character"
+           
+[1] "Incoming data: rough road leads to the stars \nчерез тернии к звездам\n"
+
+[[1]]
+[1] "Incoming query params: "
+```
+Model value is as it was returned by the `load_model` hook.  
+Content-Type was not specified so default values are passed: `mimetype=text/plain` and `charset=utf8`, so data is treated as text and passed as `character` into the hook.
+Query params are empty as `--query` option argument was not provided.
```

Comment:
```suggestion
The model value represents what is returned by the `load_model` hook.  
The content type was not specified, so the default values are passed: `mimetype=text/plain` and `charset=utf8`. In this case the data is treated as text and passed as `character` into the hook.
The query params are empty because the `--query` argument (optional) was not provided.
```